### PR TITLE
Handle exceptions when no language features detected

### PR DESCRIPTION
**Pull request by mariusfa**

### CODE REVIEW

File: languages/__init__.py [hunk 1]
```diff
@@ -12,17 +13,26 @@
 
 
 def execute(config, database, message, metadata: dict, data: io.BufferedReader):
-    lang = detect_language(metadata, data)
+    try:
+        lang = detect_language(metadata, data)
+        if not lang:
+            log.info("Could not detect language")
+            return
```
Comment (mabruras) [inline languages/__init__.py]: Is this necessary, if `detect_language` raises exception when no language features are detected?

File: languages/__init__.py [hunk 2]
```diff
@@ -12,17 +13,26 @@
...
+        log.info("Could not detect language")
 
-    save_to_db(database, message, lang)
 
 
 def detect_language(metadata, data):
     log.info('Detecting language of object')
     lang = metadata.get('lang', None) or metadata.get('language', None)
 
+    MAX_BYTE_SIZE_TO_READ = 10000
```
Comment (mabruras) [inline languages/__init__.py]: Maybe move this constant to module level, or rename to a lower case variable.

File: languages/__init__.py [hunk 3]
```diff
@@ -12,17 +13,26 @@
...
+
     if not lang:
-        lang = detect(data.read().decode())
+        stringData = data.read(MAX_BYTE_SIZE_TO_READ).decode()
```
Comment (mabruras) [inline languages/__init__.py]: Please use naming convention according to [`PEP8` - Function and Variable Names](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

File: languages/__init__.py [hunk 4]
```diff
@@ -12,17 +13,26 @@
...
+        save_to_db(database, message, lang)
+    except LangDetectException:
+        log.info("Could not detect language")
 
-    save_to_db(database, message, lang)
 
 
 def detect_language(metadata, data):
     log.info('Detecting language of object')
     lang = metadata.get('lang', None) or metadata.get('language', None)
```
Comment (mabruras) [inline languages/__init__.py]: An alternative to hard coding in `lang` and `language` could be to fetch the potential name from the configuration. Like creating a tag in the `custom` part of the plugin configuration, as in these examples:
* Faces plugin: [config](https://github.com/mabruras/sqapi-plugins/blob/master/faces/config.yml#L18) - [usage](https://github.com/mabruras/sqapi-plugins/blob/master/faces/__init__.py#L23-L24)
* Thumbnails plugin: [usage](https://github.com/mabruras/sqapi-plugins/blob/master/thumbnails/__init__.py#L21-L22)

**Review by mabruras — CHANGES_REQUESTED** [review]

**Comment by mariusfa** [comment]
Also testet a file to check for language in metadata

**Review by mabruras — APPROVED** [review]

**Pull request merged**